setup.py

```diff
@@ -1,14 +1,30 @@
-from distutils.core import setup
-from RefRed.version import str_version
-
-__package_name__ = 'RefRed'
-__version__ = str_version
-__description__ = 'Liquids Reflectrometer Data Reduction Software'
-__author__ = 'Jean Bilheux'
-__author_email__ = 'bilheuxjm@ornl.gov'
-__license__ = 'Copyright 2015-2016'
-__url__ = 'http://'
-__scripts__ = ['scripts/RefRed', 'scripts/start_refred.py']
+import os
```

Comment:
Overall, will the current change be able to `set up` RefRed successfully and guarantee a successful build on `Travis`?